{
  "path": "/documentation/cfnetwork/cfnetworkerrors/cfurlerrorunknown",
  "type": "Enumeration Case",
  "name": "CFNetworkErrors.cfurlErrorUnknown",
  "desc": "An unknown error occurred.",
  "items": [],
  "declaration": "case cfurlErrorUnknown = -998"
}